torch/fx/experimental/symbolic_shapes.py

```diff
@@ -214,7 +214,8 @@ def is_concrete_int(a: Union[int, SymInt]) -> bool:
     if isinstance(a, int):
         return True
 
-    if isinstance(a.node.expr, sympy.core.numbers.Integer):
+    # note: NestedInt is a SymInt but does not have a.node.expr defined.
+    if isinstance(getattr(a.node, "expr", None), sympy.core.numbers.Integer):
```

Comment:
Do you know where this is called?
Nested int technically "concrete" and in many(most?) cases nested_int is intended to be used where int is used. Could we want to return True instead?

Reply:
@soulitzer - I switched this to return True for NestedInts and it also fixes this issue (although the other issue with evaluation of whether a tensor is a tracing tensor still exists after changing this)